README.md

```diff
@@ -67,6 +67,48 @@ print table:keys dict
 => ["turtle" "bunny"]
 ```
 
+### Manipulating Tables
+
+If the same key is used with `table:put` more than once for the same table, the value provided to *last* call of `table:put` will be the value shown when `table:get` is used.
+Here is an example:
+
+```NetLogo
+let dict table:make
+table:put dict "a" 5
+table:put dict "a" 3
+print table:get dict "a"
+=> 3
+```
+
+Because tables are mutable, manipulating existing values should be done by calling `table:get` on a key, transforming the returned value, and then calling `table:put` to update the transformed value in the table.
+Here is an example procedure which increments a value in a table at a given key:
+
+```NetLogo
+to increment-table-value [ dict key ]
+  let i table:get dict key
```

Comment:
Should we make this example robust to the key not existing, as in the user's example?

That would be

```
let i table:get-or-default dict key 0
```

I believe.